src/simsoptpp/python_surfaces.cpp

```diff
@@ -149,6 +149,7 @@ template <typename T, typename S> void register_common_surface_methods(S &s) {
      .def("fit_to_curve", &T::fit_to_curve, py::arg("curve"), py::arg("radius"), py::arg("flip_theta") = false)
      .def("scale", &T::scale)
      .def("extend_via_normal", &T::extend_via_normal)
+     .def("extend_via_projected_normal", &T::extend_via_projected_normal)
```

Comment:
Somewhere, perhaps here so it shows up as a python docstring, can you add a comment with a few sentences explaining what `extend_via_projected_normal` does, and what its arguments mean?

Reply:
Presently these comments are not visible from python. But if you move these comments to a string inside the closing parenthesis of the `.def()` bit in the line above, then they become regular python docstrings visible to IDEs and `help()`. See e.g. `python_boozermagneticfield.cpp`